testing/testcafe/tests/dataGrid/helpers/generateDataSourceData.ts

```diff
@@ -10,3 +10,15 @@ export const getData = (rowCount: number, colCount: number): Record<string, stri
 
   return items;
 };
+export const getNumberData = (rowCount: number, colCount: number): Record<string, number>[] => {
```

Comment:
Let's add the empty line here:

```suggestion
};

export const getNumberData = (rowCount: number, colCount: number): Record<string, number>[] => {
```